bin/tester/runtimes.js

```diff
@@ -89,11 +89,20 @@ export default async function getRuntimeSpecs(url, { verbose = false } = {}) {
       );
     }
 
-    console.log(dedent`
-        This may lead to some stories not working right or getting detected by Chromatic
-        We suggest you fix the errors, but we will continue anyway..
-        ${separator}
-      `);
+    if (errors.length && !log.level.match(/silent/)) {
```

Comment:
Are we going to allow them to continue if log level is silent? That seems incorrect, maybe?

Reply:
👍